output/components.eslint.txt

```diff
@@ -16,7 +16,8 @@ The react/require-extension rule is deprecated. Please use the import/extensions
   4:2  error  Unexpected require()                                                                    global-require
 
 /home/travis/build/Talend/ui/packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js
-  48:3  error  Visible, non-interactive elements should not have mouse or keyboard event listeners  jsx-a11y/no-static-element-interactions
+  21:1  error  Line 21 exceeds the maximum line length of 100                                       max-len
```

Comment:
at least fix this one to not introduce one more eslint violation else its all good